### ISSUE TITLE
ADS-B FlightAware Unique Identifier changes after HA restart / upgrade

### ISSUE THREAD

**Issue by pstoian** [MaxWinterstein/homeassistant-addons#111]
After the Home Assistant upgrade or reboot - the ADS-B Addon is stop connecting with FlightAware servers. I have to re-claim it via FlightAware web site - and appears as a different device hence new Unique Identifier. In the last week after HA OS or HA supervisor upgrade - I have to reclaim it several times and in each instance a new ID was showing.

**Comment by MaxWinterstein** [comment]
Can't really confirm that. Do you have a `PIAWARE_FEEDER_DASH_ID` in your config? 

Might share your config (redacted of course)?

**Comment by pstoian** [comment]
Yes - the PIAWARE_FEEDER_DASH_ID is all good and has been set.  But only when I am rebooting the add one or the HA hardware host - I am loosing the connectivity to FlightAware and I noticed that I have to reclaim it and I have a different Unique Identifier. 

SERVICE_ENABLE_DUMP1090: true
SERVICE_ENABLE_PIAWARE: true
SERVICE_ENABLE_FR24FEED: true
SERVICE_ENABLE_HTTP: true
PIAWARE_FEEDER_DASH_ID: < MY FEDER ID>
FR24FEED_FR24KEY: <I need assistance to get the key>
ADSBEXCHANGE_UUID: ""
PLANEFINDER_SHARECODE: ""
HTML_SITE_LAT: "my LAT"
HTML_SITE_LON: "my LON"
HTML_DEFAULT_TRACKER: FlightAware
HTML_SITE_NAME: <my site>

**Comment by MaxWinterstein** [comment]
tried to reproduce it, but can't find an issue here. 

I am not running HA OS, so it might be related to something. As I was going to switch toward HA OS I will have a look again then. 

But for the moment, I am clueless.

**Comment by MaxWinterstein** [comment]
Closing this due to inactivity — no updates in over 2 years. I was unable to reproduce the issue at the time, and it may well have been fixed in subsequent HA OS or addon updates. If you're still experiencing this, feel free to reopen with updated logs and addon version info.
